SettingsList.py

```diff
@@ -2430,22 +2430,16 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             True: {'settings': ['shuffle_weird_egg']},
         },
     ),
-    Setting_Info(
-        name           = 'dungeon_shortcuts',
-        type           = list,
-        gui_text       = 'Dungeon Boss Shortcuts',
-        gui_type       = "MultipleSelect",
+    Combobox(
+        name           = 'dungeon_shortcuts_choice',
+        gui_text       = 'Dungeon Boss Shortcuts Choice',
```

Comment:
To keep consistency with #1485, I'd rename the `gui_text` here to 'Dungeon Boss Shortcuts Mode', and...